src/store/tests/test-data/cat-adi-part2/vehicle-checks/vehicle-checks.cat-adi-part2.action.ts

```diff
@@ -1,6 +1,18 @@
 import { createAction } from '@ngrx/store';
 import { QuestionResult, QuestionOutcome } from '@dvsa/mes-test-schema/categories/common';
 
+export const SHOW_ME_QUESTION_SELECTED = '[VehicleChecksPage] [CatADI2] Show Me Question Selected';
+export const SHOW_ME_QUESTION_OUTCOME_CHANGED = '[VehicleChecksPage] [CatADI2] Show Me Question Outcome Changed';
+export const TELL_ME_QUESTION_SELECTED = '[VehicleChecksPage] [CatADI2] Tell Me Question Selected';
+export const TELL_ME_QUESTION_OUTCOME_CHANGED = '[VehicleChecksPage] [CatADI2] Tell Me Question Outcome Changed';
+export const SHOW_ME_QUESTION_ADD_DRIVING_FAULT = '[Vehicle Checks] [CatADI2] Show Me Question Add Driving Fault';
+export const SHOW_ME_QUESTION_REMOVE_DRIVING_FAULT = '[Vehicle Checks] [CatADI2] Show Me Question Remove Driving Fault';
+export const VEHICLE_CHECKS_ADD_SERIOUS_FAULT = '[Vehicle Checks] [CatADI2] Vehicle Checks Add Serious Fault';
+export const VEHICLE_CHECKS_REMOVE_SERIOUS_FAULT = '[Vehicle Checks] [CatADI2] Vehicle Checks Remove Serious Fault';
+export const VEHICLE_CHECKS_REMOVE_DANGEROUS_FAULT = '[Vehicle Checks] [CatADI2] Vehicle Checks Remove Dangerous Fault';
+export const ADD_SHOW_ME_TELL_ME_COMMENT = '[Vehicle Checks] [CatADI2] Add Show Me / Tell Me comment';
+export const VEHICLE_CHECKS_TOGGLE = '[Vehicle Checks] [CatADI2] Vehicle Checks Completed Toggled';
```

Comment:
These will not be needed, if we need to reference the action in an effect, we just do

```ofType(ShowMeQuestionSelected),```
... and so on